src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.commands;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.*;
```

Comment:
Minor comment.
Imported classes should always be listed explicitly.

Please look at https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements